app/src/main/java/com/infomaniak/mail/utils/ConfettiUtils.kt

```diff
@@ -28,9 +29,48 @@ import com.infomaniak.lib.confetti.R as RConfetti
 
 object ConfettiUtils {
 
+    private const val EASTER_EGG_CONFETTI_TRIGGER_TAPS = 3
+    private const val EASTER_EGG_CONFETTI_TRIGGER_DELAY = 1_000L
+
     private const val EMISSION = 1_000_000.0f
 
-    fun triggerEasterEggConfetti(container: ViewGroup, matomoValue: String): Unit = with(container.context) {
+    private const val AQUA = -0xFF0001
+    private const val FUCHSIA = -0xFF01
+    private const val LIME = -0xFF0100
+    private const val MAROON = -0x800000
+    private const val NAVY = -0xFFFF80
+    private const val OLIVE = -0x7F8000
+    private const val PURPLE = -0x7FFF80
+    private const val SILVER = -0x3F3F40
+    private const val TEAL = -0xFF7F80
```

Comment:
If you need to specify hex colors, don't specify hexadecimal colors as negative numbers this makes it unintelligible when you try to read the numbers, use the standard notation `0xAARRGGBB`.

That being said, here, you should use a resources to store these values inside a color array